static/app/views/settings/project/projectKeys/details/keySettings.tsx

```diff
@@ -36,14 +42,50 @@ type Props = {
   };
 };
 
-export function KeySettings({api, onRemove, data, organization, params}: Props) {
+export enum DynamicSDKLoaderOption {
+  HAS_DEBUG = 'hasDebug',
+  HAS_PERFORMANCE = 'hasPerformance',
+  HAS_REPLAY = 'hasReplay',
```

Comment:
I wonder if we should log an error to Sentry if we detect that there is a key defined in the frontend that the backend is no longer sending.

Thoughts?

Reply:
hm yes we can do that 😉 will add it in a follow-up